client/src/components/employer-form/Review.js

```diff
@@ -73,7 +74,32 @@ export const Review = ({ handleEditClick, isDisabled }) => {
               <Grid container alignItems="center" justify="space-between" spacing={2}>
                 <Grid item>
                   <Typography variant="subtitle1">
-                    3. HCAP Request
+                    3. Site Type and Size
+                  </Typography>
+                </Grid>
+                {!isDisabled && (
+                  <Grid item>
+                    <Button
+                      startIcon={<EditIcon />}
+                      fullWidth={false}
+                      size="small"
+                      onClick={() => handleEditClick(2)}
```

Comment:
I think it should redirect to the section 3.

Reply:
GREAT catch!